src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```

Comment:
Ple\se use correct formatting
```suggestion
  const { 
    name, 
    age, 
    sex, 
    isMarried, 
    partnerName 
  } = person;
```